modules/recieve_statustxt.py

```diff
@@ -0,0 +1,63 @@
+import time
+from pymavlink import mavutil
+from modules.common.data_encoding.message_encoding_decoding import decode_bytes_to_position_global
+from modules.common.data_encoding.metadata_encoding_decoding import decode_metadata
+from modules.common.data_encoding.worker_enum import WorkerEnum
+from modules.common.kml.kml_conversion import positions_to_kml
+from pathlib import Path
+
+CONNECTION_ADDRESS = "tcp:localhost:14550"
+DELAY = 1
```

Comment:
Our convention is to place 2 spaces after global constants. Also, for your script you would not need a `DELAY` variable, since `recv_match` is blocking

Reply:
Done :)